docs/spec/index.html

```diff
@@ -1,740 +1,952 @@
-            SHOULD, MUST, or MAY be included with each traceable presentation.
-          </p>
-        </section>
       </section>
-
-
     </section>
-
-    <section id="terminology" class="informative">
-      <h2>Terminology</h2>
+  </section>
+
+  <section class="informative">
+    <h2>Interoperability Testing</h2>
+    <p>
+      Participating platforms are proving actual interoperability by
+      enrolling in continuous integration tests. These tests are based
+      on Postman collections which orchestrate the interaction of
+      participating parties, such as `Credential Issuance`, `Credential
+      Verification`, and `Presentation Exchange`. These Postman
+      collections are described with a set of
+      <a
+        href="https://github.com/w3c-ccg/traceability-interop/tree/main/docs/tutorials">tutorials</a>
+      that show how to run the tests against a particular implementation.
+    </p>
+    <p>
+      An interoperability report is continuously created from the
+      results of executing these collections, targeting the
+      participating platforms. In cases of multi-party interactions
+      (such as party A making a verifiable presentation to party B), all
+      combinations of platform interchange are executed. The
+      Interoperability report is executed by a GitHub Action on the
+      Traceability Interoperability GitHub repository:
+      <a
+        href="https://github.com/w3c-ccg/traceability-interop/actions/workflows/interoperability-report.yml">interoperability-report.yml</a>
+    </p>
+    <p>
+      In each test, the GitHub Action runner carries out a number of
+      invocations, acting on behalf of varying parties taking different
+      roles (`issuer`, `holder`, and `verifier`), in turn invoking the
+      participating platform.
+    </p>
+    Below is an example of calls made by the runner as it is executing a
+    DIDAuth Presentation Exchange flow:
+    <ul>
+      <li>Runner calls `/credentials/issue` on Party A's platform, representing
+        Party A acting as `issuer`.</li>
+      <li>Runner calls `/presentations/available` on Party B's platform,
+        representing Party A acting as `holder`.</li>
+      <li>Runner calls `/presentations/prove` on Party A's platform,
+        representing Party A acting as `holder`.</li>
+      <li>
+        Runner calls `/presentations/submissions` on Party B's platform,
+        representing Party A acting as `holder`.
+      </li>
+      <li>Runner calls `/credentials/verify` on Party B's platform, representing
+        Party B acting as `verifier`.</li>
+    </ul>
+  </section>
+
+  <section class="informative">
+    <h2>OAS Implementation</h2>
+    <p>
+      The Traceability Interoperability
+      <a href="https://w3id.org/traceability/interoperability/openapi">Open API
+        Specification</a>
+      is designed to be easily implemented and used to access conforming
+      platforms.
+    </p>
+    <p>
```

Comment:
We might want to provide some links related to code gen here... such as:

https://www.npmjs.com/package/openapi-typescript-codegen

Reply:
Created #503